ios/Runner/AppDelegate.swift

```diff
@@ -143,12 +138,15 @@ func MissingArgumentError(message: String, details: Any?) -> FlutterError {
     func startSite(call: FlutterMethodCall, result: @escaping FlutterResult) {
         guard let args = call.arguments as? Dictionary<String, String> else { return result(NoArgumentsError()) }
         guard let id = args["id"] else { return result(MissingArgumentError(message: "id is a required argument")) }
+        
 #if targetEnvironment(simulator)
         let updater = self.sites?.getUpdater(id: id)
+        activeSite = updater?.getSite()
```

Comment:
Is this being used for something?

Reply:
This is specific to enabling the simulator, which lacks network extension support. Only used in dev to avoid having to pick up another device.

Reply:
And deleted, nice catch!